src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/config/WebClientConfiguration.kt

```diff
@@ -70,6 +72,15 @@ class WebClientConfiguration(
   }
 
   private fun createAuthorizedWebClient(clientManager: OAuth2AuthorizedClientManager, baseUrl: String): WebClient {
```

Comment:
i think this would be neater as a single method with default parameters i.e. 

```
  private fun createAuthorizedWebClient(
    clientManager: OAuth2AuthorizedClientManager,
    baseUrl: String,
    readTimeoutSeconds: Int = readTimeoutSeconds,
    connectTimeoutSeconds: Long = connectTimeoutSeconds,
  ): WebClient
```

Reply:
there is some name shadowing going on here too which would be good to avoid. perhaps we could call the existing configuration vars `defaultReadTimeoutSeconds` etc